tests/e2e/lifecycle/mcg/test_mcg_namespace_lifecyle_crd.py

```diff
@@ -10,6 +10,8 @@
     skipif_aws_creds_are_missing,
     skipif_managed_service,
     red_squad,
+    mcg,
+    rgw,
```

Comment:
@sagihirshfeld Since we are not using `rgw` mark here, could you please remove it

Reply:
Thanks for spotting it - fixed